src/pfe/portal/modules/Templates.js

```diff
@@ -71,9 +88,15 @@ module.exports = class Templates {
       }
     }
 
-    await Promise.all(Object.values(repos).map(async function getTemplates(repository) {
+    await Promise.all(Object.values(repos).map(async function getTemplatesFromRepo(repository) {
 
-      let repositoryUrl = new URL(repository.url);
+      let repositoryUrl;
+      try {
+        repositoryUrl = new URL(repository.url);
+      } catch (error) {
```

Comment:
Good catch here! (pun-intended)

Reply:
haha, yea found it while adding unit tests